### ISSUE TITLE
Create a design document for including room/level-specific NPCs

### ISSUE THREAD

**Issue by shayna-k** [uchicago-cs/chiventure#519]
Openworld is considering creating various rooms or levels. We would think about what kind of NPCs would be present in which room/level. We would also think of including some battle functionality with the NPCs of the specific room/level.

**Comment by shayna-k** [comment]
After coordinating with rpg-battle and rpg-openworld, we decided that to start off this feature, our team would independently write some sample code at first.
Once we have a foundation, we would come back together and collaborate regarding any changes, additions or deletions we find necessary in our implementation.
We will close this issue for this sprint as we came to an informal understanding of what would be required from this feature, from other wikis, design documents and code. We shall open a new issue in the future when it comes to discussing the implementation and collaborating with rpg-battle and rpg-openworld.
Our team shall open an issue regarding the sample implementation for the third sprint.
